src/lib/mongoBackend/mongoUpdateSubscription.cpp

```diff
@@ -355,14 +355,46 @@ static void setCondsAndInitialNotify
       status = subOrig.hasField(CSUB_STATUS)? getStringFieldF(subOrig, CSUB_STATUS) : STATUS_ACTIVE;
     }
 
-    std::string url;
+    HttpInfo httpInfo;
     if (subUp.notificationProvided)
     {
-      url = subUp.notification.httpInfo.url;
+      httpInfo = subUp.notification.httpInfo;
```

Comment:
Do we really need this copy here?
I mean, couldn't subUp.notification.httpInfo be used directly, without going via httpInfo ?